src/pybind/mgr/dashboard/frontend/src/app/ceph/block/block.module.ts

```diff
@@ -198,6 +200,16 @@ const routes: Routes = [
         ]
       }
     ]
+  },
+  // Nvme-oF/TCP
+  {
+    path: 'nvmeof',
+    canActivate: [FeatureTogglesGuardService],
+    data: { breadcrumbs: 'Nvme-oF' },
```

Comment:
I think the suggestion was to call it NVMe/TCP on one of the presentation so let's do it like it.
```suggestion
    data: { breadcrumbs: 'NVMe/TCP' },
```